spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianIframeContentServlet.java

```diff
@@ -34,7 +34,7 @@ protected AtlassianIframeContentServlet(LoginUriProvider loginUriProvider,
 
 
     @Override
-    protected String prepareIndexHtml(String indexHtml, RequestProperties props) throws IOException {
+    protected String prepareIndexHtml(String indexHtml, RequestProperties props) {
```

Comment:
This and similar changes in `AtlassianAppServlet` seemed like a good idea first, but then I realized that now subclasses can no longer throw an IOException when overriding this method.

That's because in Java the signatures cannot be widened by subclasses.

If we remove the `IOException` here we should do the same in `AppServlet`, which initially declares this method. I'm inclined in reverting this change (and the same in 'AtlassianAppServlet'), though, to leave this flexibility to subclasses.

Thoughts?

Reply:
After some discussion I decided to revert the changes, restoring the previous signatures.